src/app/pages/main-page/components/header/header.component.html

```diff
@@ -0,0 +1,30 @@
+<header>
+  <div class="header__name">
+    <a href="#"> Cudo-shop </a>
+  </div>
```

Comment:
If we click on a link with `href="#"`, it will trigger reloading page. We don't want it. Please, replace it into element with `click` metod like example below:
```suggestion
<header>
  <div class="header__name">
  <button (buttonClicked)="buttonClickedGoToHome.emit()">Cudo-shop</button>
  </div>
```

Reply:
done